function_exercises.js

```diff
@@ -0,0 +1,271 @@
+console.log(' ');
```

Comment:
I think you definitely understood how to use functions and scoping. No major errors or feedbacks other than what I mentioned below. Good job and keep up the good work!